docs/Modules/README.md

```diff
@@ -4,3 +4,6 @@ In order to create and add a new package to this project/repo, please follow the
 
 1. Make sure your new package name is prefixed with `@deriv/`.
 2. Make sure the Webpack config has `IS_RELEASE` flag, as well as conditional source-maps based on "if IS_RELEASE".
+3. To link the new package with the previous packages that we already have, you need to run `npm run bootstrap` to link it with the others.
```

Comment:
```suggestion
3. To link the new package with the previous packages that we already have, you need to run `npm run bootstrap:dev` to link it with the others.
```
We should run `npm run bootstrap:dev` to install/update/remove a package, so that it updates the same in package-lock.json file

Reply:
Thank u @balakrishna-binary for your revision.